### ISSUE TITLE
Feature Request: Create Folders

### ISSUE THREAD

**Issue by OSMmapper** [davidhealey/waistline#624]
Hello and first of all thank you for the app.
Would it be possible to create different folders in the future?
In these you could store food and drinks by eg by manufacturer or similar (McDonalds, Burger King, canned food, supermarket...) in individual folders.
Meanwhile, my list is simply already relatively long and I must therefore always use the search function.
This would certainly improve with multiple folders.

Thanks

**Comment by davidhealey** [comment]
There is a category feature which might help.

**Comment by EmilJunker** [comment]
I agree, the food categories were added precisely with this use case in mind. I don't see why we would need to add folders.

**Comment by OSMmapper** [comment]
You mean by categories the two icons (tree and 24)? 
That is also a possibility. 
But then it should be possible to customize the categories and/or add as many more as you want. 
Must not necessarily be an icon. Text would be enough.

**Comment by EmilJunker** [comment]
> But then it should be possible to customize the categories and/or add as many more as you want.

You can do that under `Settings > Foods, Meals, Recipes > Labels and Categories`
